yo/generator/page/templates/page.js

```diff
@@ -0,0 +1,60 @@
+
+module.exports = function setupRoute(/* dependencies from boring */ boring) {
+
+  const {
+    logger,
+    config,
+    decorators,
+  } = boring;
+
+  const {
+    endpoint,
+    post,
+    get,
+    entrypoint,
+  } = decorators.router;
+
+  // signals to boring this class has HTTP decortors as class methods
+  @endpoint()
+  class <%= className %> {
+
+    // The @get decorator is the primary mechanism to setup a route in boring
+    @post('<%= path %>data.json')
+    serve_<%= pageName %>_data(req, res) {
+      logger.info('Severing JSON for <%= pageName %>');
+      res.json({
+        msg: 'hello world, good to meet you ' + req.body.name
+      });
+    }
+
+    @get('<%= path %>')
+    @entrypoint('client/pages/<%= pageName %>/entrypoint.js')
+    serve_<%= pageName %>_get(req, res) {
+      logger.info('Severing JSON for <%= pageName %>');
+
+
+      const links = res.locals.pageInjections.headLinks || [];
+      const scripts = res.locals.pageInjections.bodyEndScripts || [];
+
+      res.send(`
+    <!DOCTYPE html>
+      <html style={{ height: '100%' }}>
+        <head>
+          ${links.map(link => {
+            return `<link rel="stylesheet" href="${link.href}" />`;
+          }).join('\n')}
+        </head>
+        <body style={{ height: '100%', padding: '0px', margin: '0px' }}>
+          <div style={{ width: '100%', height: '100%' }} id="root">
+
+          </div>
+          ${scripts.map(script => {
+            return `<script src="${script.src}"></script>`;
```

Comment:
It doesn't look like this is doing much, but this is most of the value boring brings even to vanilla JS development (not trying to claim a million other things also bring the same value, just pointing out boring care do vanillia app dev).

* Automatically sets up an `entrypoint` in webpack then magically gives you a value of `script.src` that is served out of `webpack-dev-middleware` (embedded in`boring`).  This "just works" regardless if the script is bundled by dev middleware or an actual prod build. 
* Will bundle any modules you import from entrypoint
* Setup webpack hot module reload to reload client side files when a developer changes code.